data/data/agent/systemd/units/create-cluster-and-infraenv.service.template

```diff
@@ -3,7 +3,6 @@ Description=Service that creates initial cluster and infraenv
 Wants=network-online.target assisted-service.service
 PartOf=assisted-service-pod.service
 After=network-online.target assisted-service.service
-ConditionPathExists=/etc/assisted-service/node0
```

Comment:
I may be wrong, but I think that the dependencies on assisted-service prevented that, given that the path unit triggered the ancestor assisted-service-pod (but with that approach it would have been better to remove the install section on all the dependent services I guess).
Surprinsigly, the `ConditionPathExists=/etc/assisted-service/node0` is not present on assisted-service (even though its BindsTo+After on assisted-service-pod makes the dependency pretty strong)